api/overdrive.py

```diff
@@ -1109,7 +1118,6 @@ class RecentOverdriveCollectionMonitor(OverdriveCirculationMonitor):
     """Monitor recently changed books in the Overdrive collection."""
 
     SERVICE_NAME = "Reverse Chronological Overdrive Collection Monitor"
-    INTERVAL_SECONDS = 60
```

Comment:
This is a constant that we haven't used for a long time; I'm just cleaning it up here.